coresimd/powerpc/altivec.rs

```diff
@@ -868,6 +882,28 @@ mod tests {
         assert_eq!(d, vec_madds(a, b, c).into_bits());
     }
 
+    #[simd_test(enable = "altivec")]
+    unsafe fn test_vec_mradds() {
+        let a: vector_signed_short = i16x8::new(
+            0 * 256,
+            1 * 256,
+            2 * 256,
+            3 * 256,
+            4 * 256,
+            5 * 256,
+            6 * 256,
+            7 * 256,
+        ).into_bits();
+        let b: vector_signed_short =
+            i16x8::new(256, 256, 256, 256, 256, 256, 256, 256).into_bits();
+        let c: vector_signed_short =
+            i16x8::new(0, 1, 2, 3, 4, 5, 6, i16::max_value() - 1).into_bits();
+
+        let d = i16x8::new(0, 3, 6, 9, 12, 15, 18, i16::max_value());
+
+        assert_eq!(d, vec_madds(a, b, c).into_bits());
```

Comment:
Is this supposed to be vec_madds?

Reply:
Good catch!
No, it should be an r that got forgotten...